pages/alpha/index.tsx

```diff
@@ -192,13 +189,16 @@ const AlphaPage = ({ address }: { address: string }) => {
       const nextNextEvent = feedEvents[i + 2];
       feedItems.push(event);
 
-      if (shouldAggregate(event, nextEvent, nextNextEvent)) {
+      // Single person aggregate
+      if (shouldAggregateFollows(event, nextEvent, nextNextEvent)) {
+        // remove the item that would start the aggregation
+        feedItems.pop();
         // const eventsAggregated: FeedQueryEvent[] = feedEvents
         //   .slice(i)
         //   .filter((fe, i, arr) => shouldAggregate(fe, arr[i + 1]));
         const eventsAggregated: FeedQueryEvent[] = [feedEvents[i]];
         let j = i + 1;
-        while (shouldAggregate(feedEvents[j], feedEvents[j + 1], feedEvents[j + 2])) {
+        while (shouldAggregateFollows(feedEvents[j], feedEvents[j + 1], feedEvents[j + 2])) {
```

Comment:
Are `j`, `j+1` and `j+2` guaranteed to exist? I see in already-committed-code above that there's references to `i+1` and `i+2`, but it's not clear to me those are safe either. And it seems... dirty to try to reference them if they dont.